src/Views.elm

```diff
@@ -2,14 +2,16 @@ module Views exposing (..)
 
 import Html exposing (Html, text, div)
 import Html.Attributes exposing (class)
-import Types exposing (Msg, Model, Route(SilencesRoute, AlertsRoute, StatusRoute))
+import Types exposing (Msg, Model, Route(..))
 import Utils.Types exposing (ApiResponse(..))
-import Utils.Views exposing (error, loading, notFoundView)
-import Translators exposing (alertTranslator, silenceTranslator)
-import Silences.Views
-import Alerts.Views
-import Status.Views
-import NavBar.Views exposing (appHeader)
+import Utils.Views exposing (error, loading)
+import Views.SilenceList.Views
```

Comment:
Not a fan of deep nesting. But if we choose to go for this, we can at least rename the imports `import Views.SilenceList.Views as SilenceList`.